fixtures/cameo/zenit-z120.json

```
@@ -0,0 +1,422 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Zenit Z120",
+  "shortName": "Z120",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["EP", "Ken Harris", "Luke Nelson"],
+    "createDate": "2024-01-06",
+    "lastModifyDate": "2024-01-06"
```

```suggestion
    "createDate": "2024-01-08",
    "lastModifyDate": "2024-01-08"
```

Or whatever day it is by then :sweat_smile: